plugins/source/azure/resources/services/costmanagement/view_queries_fetch.go

```diff
@@ -0,0 +1,40 @@
+package costmanagement
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/costmanagement/armcostmanagement"
+	"github.com/cloudquery/cloudquery/plugins/source/azure/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func fetchViewQueries(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
+	cl := meta.(*client.Client)
+	item := parent.Item.(*armcostmanagement.View)
+	if item.Properties == nil {
+		return nil
+	}
+
+	svc, err := armcostmanagement.NewQueryClient(cl.Creds, cl.Options)
+	if err != nil {
+		return err
+	}
+
+	b, err := json.Marshal(item.Properties.Query)
+	if err != nil {
+		return err
+	}
+	var qd armcostmanagement.QueryDefinition
+	if err := json.Unmarshal(b, &qd); err != nil {
```

Comment:
Types are not compatible (`ReportConfigDefinition` vs. `QueryDefinition`) so instead of doing a deep conversion we'll use their marshal/unmarshal methods instead.